build-tools/create-vsix/create-vsix.csproj

```diff
@@ -25,6 +25,8 @@
     <ExtensionInstallationFolder>Xamarin\Xamarin.Android.Sdk</ExtensionInstallationFolder>
     <OutputPath>bin\$(Configuration)</OutputPath>
   </PropertyGroup>
+  <PropertyGroup Condition=" '$(Configuration)' == 'Debug' " />
+  <PropertyGroup Condition=" '$(Configuration)' == 'Release' " />
```

Comment:
Shouldn't we then add the usual Debug/Release property groups? Like:
```
  <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Debug|AnyCPU' ">
    <DebugSymbols>true</DebugSymbols>
    <DebugType>full</DebugType>
    <Optimize>false</Optimize>
    <OutputPath>$(UtilityOutputFullPath)</OutputPath>
    <DefineConstants>DEBUG;</DefineConstants>
    <ErrorReport>prompt</ErrorReport>
    <WarningLevel>4</WarningLevel>
    <Externalconsole>true</Externalconsole>
  </PropertyGroup>
  <PropertyGroup Condition=" '$(Configuration)|$(Platform)' == 'Release|AnyCPU' ">
    <DebugType>full</DebugType>
    <Optimize>true</Optimize>
    <OutputPath>$(UtilityOutputFullPath)</OutputPath>
    <ErrorReport>prompt</ErrorReport>
    <WarningLevel>4</WarningLevel>
    <Externalconsole>true</Externalconsole>
  </PropertyGroup>
```

Reply:
Since this project doesn't build C# code, I made it the same as some of the other projects like:
https://github.com/xamarin/xamarin-android/blob/13a692511ad77c1494c2e9aaff84d7d60efb0de0/src/r8/r8.csproj#L9-L10

@radekdoulik after looking at this more, I don't think this change is going to help the build problem  we are having downstream in monodroid.

I think this _only_ fixes an IDE issue in Visual Studio, from the screenshot above.